components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/template/APITemplateBuilderImpl.java

```diff
@@ -109,14 +152,57 @@ public void addHandler(String handlerName, Map<String, String> properties) {
 		addHandlerPriority(handlerName, properties, handlers.size());
 	}
 
-	public void addHandlerPriority(String handlerName, Map<String, String> properties, int priority) {
+	public void addHandlerPriority(String handlerName, Map<String, String> properties,
```

Comment:
At the committing time recommended max column length was 100.
 Fixed in commit bde62335f3cd69ee2a017a3dcb55234ce3262419 of PR https://github.com/wso2/carbon-appmgt/pull/153